pkg/terraform/executor.go

```diff
@@ -211,22 +213,50 @@ func (ex *Executor) execute(verbose bool, args ...string) error {
 		wg.Add(1)
 
 		go tailFile(p, done, &wg)
+	} else {
+		fmt.Printf("\nYou can find the logs in %q\n", p)
 	}
 
 	wg.Wait()
 
 	s, err := ex.Status(pid)
 	if err != nil {
+		if !verbose {
+			showError(p, noOfLinesOnError)
+		}
 		return fmt.Errorf("failed checking execution status: %w", err)
 	}
 
 	if s != ExecutionStatusSuccess {
+		if !verbose {
+			showError(p, noOfLinesOnError)
+		}
 		return fmt.Errorf("executing Terraform failed, check %s for details", p)
 	}
 
 	return nil
 }
 
+func showError(path string, noOfLines int) {
+	data, err := ioutil.ReadFile(path)
+	if err != nil {
+		fmt.Printf("error reading file: %v", err)
+		return
+	}
+	text := string(data)
+
+	lines := strings.Split(text, "\n")
+	offset := len(lines) - noOfLines - 1
```

Comment:
```suggestion
	offset := len(lines) - noOfLines - 1 //nolint:gomnd
```

Reply:
Actually, why 1? :smile:

Reply:
Oh that is to adjust the numbers. So we count from 1 and computer counts from 0 so this negation makes that adjustment.

Reply:
Added comment.